homeassistant/components/launch_library/__init__.py

```diff
@@ -27,7 +27,10 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 
     async def async_update():
         try:
-            return await launches.upcoming_launches()
+            return {
+                UPCOMING_LAUNCHES: await launches.upcoming_launches(),
+                STARSHIP_EVENTS: await launches.starship_events(),
+            }
```

Comment:
Let's add a TypedDict for this so we can reuse the typing in other modules

Reply:
Ignore my comments, mypy gave me the answer 😄

Reply:
yeah, once you use a TypedDict we would use the string directly to access it